doc/classes/ProjectSettings.xml

```diff
@@ -937,6 +937,11 @@
 		<member name="input/ui_text_caret_word_right.macos" type="Dictionary" setter="" getter="">
 			macOS specific override for the shortcut to move the text cursor forward one word.
 		</member>
+		<member name="input/ui_text_clear_carets_and_selection" type="Dictionary" setter="" getter="">
+			If there's only one active caret active with a selection, clears the selection.
+			In case there's more than one caret active, removes the secondary carets and clears selections.
```

Comment:
```suggestion
			In case there's more than one caret active, removes the secondary carets and clears their selections.
```
I believe it wouldn't clear the selection of the primary caret?

Reply:
Well spotted, the primary caret keeps its selection like in VSCode. I'll re-word that.

Reply:
Now it's correct 👍